mockData/mockListings.js

```diff
@@ -0,0 +1,30 @@
+const mockListings = [
+  {
+    name: "BBQ Grill",
+    price: 50,
+    imageSource: "./grill.jpg",
+    category: 'home'
+  },
+  {
+    name: "Stroller",
+    price: 50,
+    imageSource: "./grill.jpg",
+    category: 'baby'  
+  },
+  {
+    name: "Coffee Table",
+    price: 50,
+    imageSource: "./grill.jpg",
+    category: 'furniture'  
+  },
+  {
+    name: "Fridge",
+    price: 50,
+    imageSource: "./grill.jpg",
+    category: 'home'  
+  }
+
```

Comment:
Thanks for mocking out this data. I think we also want to include the charity name and rating here.